tests/src/system/basic/WskBasicTests.scala

```diff
@@ -477,15 +477,15 @@ class WskBasicTests
                 activation =>
                     activation.response.result shouldBe Some(dynamicParams.toJson)
                     activation.duration shouldBe 0L // shouldn't exist but CLI generates it
-                    activation.end shouldBe Instant.EPOCH.toEpochMilli // shouldn't exist but CLI generates it
+                    activation.end shouldBe Instant.ofEpochMilli(0L) // shouldn't exist but CLI generates it
```

Comment:
why dos the comment say shouldn't exist but CLI generates it - should the CLI stop generating it when this schema stripping is done?